SCClassLibrary/Common/Audio/Splay.sc

```diff
@@ -1,18 +1,33 @@
+LevelComp {
+	*new { |levelComp, rate, n|
+		if (levelComp == true) {
+			^if(rate == \audio) {
+				// ar default is equal power for backwards compatibility
+				n.reciprocal.sqrt
+			} {
+				// kr default is equal amplitude
+				n.reciprocal
+			}
+		};
+		// if false, level is untouched
+		if (levelComp == false) { ^1 };
+
+		// if a number or control signal,
+		// scale by exponent: 0 is no change,
+		// 0.5 is square root = equal power
+		// 1.0 is level / numchans = equal amplitude
+		(n.reciprocal ** levelComp.clip(0.0, 1.0));
```

Comment:
leaving out semicolon for return value is nice. Extra parentheses are not needed

Reply:
done, and added the missing return ^!